server/server.js

```diff
@@ -118,3 +118,12 @@ app.get('/server/status', (req, res) => {
   const outputJson = { status: 'True' };
   res.status(200).send(outputJson);
 });
+
+app.get('*', (req, res) => {
+  res.render(path.join(__dirname, '..', 'client', 'page_404'), {
+    header: 'Oops...',
+    // eslint-disable-next-line object-shorthand
+    sidebar: sidebar,
```

Comment:
line should just be `sidebar,`  to not have the eslint error to occur. New in ES6: See https://alligator.io/js/object-property-shorthand-es6/